File: Drools/src/main/java/org/meia/model/Question.java
package org.meia.model;

public class Question {
    private int id;
    private String question;
    private String answer;
    private String questionType;

    public Question(int id, String question, String  questionType) {
        this.id = id;
        this.question = question;
        this.setQuestionType(questionType);
    }

    public Question() {
        answer =null;
        // Deixa os atributos não inicializados
    }

    public String getQuestionType() {
        return questionType;
    }

    public int getId() {
        return id;
    }

    public String getQuestion() {
        return question;
    }

    public String getAnswer() {
        return answer;
    }

    public void setAnswer(String answer) {
        this.answer = answer;
    }

    @Override
    public String toString() {
        return answer;
    }


    public void setQuestionType(String questionType) {
        this.questionType = questionType;
    }
}

File: Spa/src/app/domain/question.ts
export class Question {

    private id : number;

    public question: string;

    public answer : string | null;

    constructor(id : number, question: string){
        this.id = id;
        this.question = question;
        this.answer = null;
    }

 
    public getId(): number {
        return this.id;
    }
  
    public getQuestion(): string {
        return this.question;
    }
  
    public getAnswer(): string | null {
        return this.answer;
    }

    public setAnswer(answer: string): void {
        this.answer = answer;
    }

    
}

